apps/wakunode2/app.nim

```diff
@@ -31,6 +31,12 @@ import
   ../../waku/v2/protocol/waku_dnsdisc,
   ../../waku/v2/protocol/waku_enr,
   ../../waku/v2/protocol/waku_discv5,
+  ../../waku/v2/protocol/waku_peer_exchange,
```

Comment:
LGTM! We may be able to delete these imports from the wakunode2.nim at some point in future PR :)

Reply:
Yes, that's the idea 😁